enigma-swing/src/main/java/org/quiltmc/enigma/gui/node/ClassSelectorClassNode.java

```diff
@@ -45,8 +47,10 @@ public void reloadStats(Gui gui, ClassSelector selector, boolean updateIfPresent
 		SwingWorker<ClassSelectorClassNode, Void> iconUpdateWorker = new SwingWorker<>() {
 			@Override
 			protected ClassSelectorClassNode doInBackground() {
-				if (generator.getResultNullable() == null || updateIfPresent) {
-					generator.generateForClassTree(ProgressListener.none(), ClassSelectorClassNode.this.getObfEntry(), false);
+				if (generator.getResultNullable() == null && generator.getOverallProgress() == null) {
+					generator.generate(new ProgressListener() {}, EnumSet.allOf(StatType.class), false);
+				} else if (updateIfPresent) {
+					generator.generateForClass(new ProgressListener() {}, ClassSelectorClassNode.this.getObfEntry(), false);
```

Comment:
Why aren't these `ProgressListener.createEmpty()`?

Reply:
because I forgot to change them!